### PR TITLE
feat: add websocket send timeout

### LINKED ISSUE #64: Add a configuration for the send timeout.

### PR DESCRIPTION
**Pull request by denopink**

### CODE REVIEW

**Comment by denopink** [comment]
close #64

File: cmd/xmidt-agent/config.go [hunk 1]
```diff
@@ -304,6 +306,7 @@ var defaultConfig = Config{
 		FetchURLTimeout:       30 * time.Second,
 		PingInterval:          30 * time.Second,
 		PingTimeout:           90 * time.Second,
+		SendTimeout:           90 * time.Second,
```
Comment (denopink) [inline cmd/xmidt-agent/config.go:322]: @schmidtw was a good default for `SendTimeout`?

**Review by piccione99 — APPROVED** [review]

**Review by piccione99 — COMMENTED** [review]
this is covered in the other commit?

**Comment by denopink** [comment]
@piccione99 I'm not sure which commit you're referring to 🙂

**Review by schmidtw — APPROVED** [review]

**Pull request merged**